### PR TITLE
Fixed uppercase url

### PR DESCRIPTION
**Pull request by ghost**
This pull request fixes issue with darthsim's imgproxy uppercase extention "Invalid URL" issue.

### CODE REVIEW

File: src/Url.php [hunk 1]
```diff
@@ -58,6 +58,7 @@ public function unsignedPath(): string
         $enlarge = (string)(int)$this->enlarge;
         $encodedUrl = rtrim(strtr(base64_encode($this->imageUrl), '+/', '-_'), '=');
         $ext = $this->extension ?: $this->resolveExtension();
+        $ext = $ext ? mb_strtolower($ext) : null;
```
Comment (crocodile2u) [inline src/Url.php:61]: Why `mb_strtolower()`? SImple `strtolower()` will do: imgproxy only supports a limited set of extensions and all of them use latin letters.

Comment (crocodile2u) [reply src/Url.php:61]: Also, there's no test

Comment (windstep) [reply src/Url.php:61]: Will fix in a day

**Comment by crocodile2u** [comment]
This has been resolved in a slightly different manner. Unit test provided, as well.

**Pull request closed without merging**